app/src/main/java/com/infomaniak/mail/ui/MainViewModel.kt

```diff
@@ -627,7 +621,7 @@ class MainViewModel @Inject constructor(
             with(ApiRepository.rescheduleDraft(resource, scheduleDate)) {
                 if (isSuccess()) {
                     val scheduledDraftsFolderId = folderController.getFolder(FolderRole.SCHEDULED_DRAFTS)!!.id
-                    refreshFoldersAsync(currentMailbox.value!!, listOf(scheduledDraftsFolderId))
+                    refreshFoldersAsync(currentMailbox.value!!, ImpactedFolders(mutableSetOf(scheduledDraftsFolderId)))
```

Comment:
We could add a constructor with only one Id and compute the mutableSet there to avoid duplicated it all the time except once